frontend/src/routes/[[lang=locale]]/candidate/(protected)/questions/+page.svelte

```diff
@@ -1,38 +1,217 @@
+          {#each categoryQuestions as question}
+            <!-- Question has been answered -->
+            {#if answers?.[question.id]}
+              <div class="pb-20 pt-20">
+                <div class="text-accent">
+                  {question.category}
+                </div>
+
+                <Expander title={question.text ?? ''} variant="question">
+                  {question.info}
+                </Expander>
+
+                <div class="pt-10">
+                  <!-- This gives empty form label error from Wave Extension for every empty dot, but fix should come from LikertResponseButton -->
+                  <LikertResponseButtons
+                    name={question.id}
+                    mode="display"
+                    options={question.values}
+                    selectedKey={answers[question.id].key} />
+
+                  {#if translate(answers[question.id].openAnswer) !== ''}
+                    <div class="pt-10">
+                      <QuestionOpenAnswer
+                        >{translate(answers[question.id].openAnswer)}</QuestionOpenAnswer>
+                    </div>
+                  {/if}
+
+                  <div class="flex justify-center py-20">
+                    <Button
+                      text={getAnsweredButtonText().text}
+                      href={$getRoute({route: Route.CandAppQuestions, id: question.id})}
+                      icon={getAnsweredButtonText().icon}
+                      iconPos="left"></Button>
+                  </div>
+                </div>
+                {#if categoryQuestions[categoryQuestions.length - 1] !== question}
+                  <hr class="mt-40" />
+                {:else}
+                  <div class="mb-40" />
+                {/if}
+              </div>
+
+              <!-- Question not yet answered -->
+            {:else}
+              <div class="pt-40">
+                <div class="text-accent">
+                  {question.category}
+                </div>
+
+                <Expander title={question.text ?? ''} variant="question" titleClass="text-warning">
+                  {question.info}
+                </Expander>
+
+                <!-- Navigate to unsanswered question -->
+                {#if dataEditable}
+                  <a
+                    class="flex justify-center py-20"
+                    href={$getRoute({route: Route.CandAppQuestions, id: question.id})}>
+                    <Button
+                      text={$t('candidateApp.questions.answerButton')}
+                      class="w-full max-w-md bg-base-300" />
+                  </a>
+                {:else}
+                  <p class="p-10">{$t('candidateApp.questions.notAnswered')}</p>
+                {/if}
+                {#if categoryQuestions[categoryQuestions.length - 1] !== question}
+                  <hr class="mt-40" />
+                {:else}
+                  <div class="mb-40" />
```

Comment:
Could the margin be assigned to some other existing element? Another option is to define more `padding-bottom` to the parent element. Having an empty div with margin is suspicious.